lab_5_scraper/scraper.py

```diff
@@ -189,6 +337,37 @@ def unify_date_format(self, date_str: str) -> datetime.datetime:
         Returns:
             datetime.datetime: Datetime object
         """
+        tm, dt = date_str.split(", ")
+        di, mi, yi = map(str, dt.split(" "))
+        d = int(di)
+        y = int(yi)
+        m = 4
+        if mi == 'января':
+            m = 1
+        if mi == 'февраля':
+            m = 2
+        if mi == 'марта':
+            m = 3
+        if mi == 'апреля':
+            m = 4
+        if mi == 'мая':
+            m = 5
+        if mi == 'июня':
+            m = 6
+        if mi == 'июля':
+            m = 7
+        if mi == 'августа':
+            m = 8
+        if mi == 'сентября':
+            m = 9
+        if mi == 'октября':
+            m = 10
+        if mi == 'ноября':
+            m = 11
+        if mi == 'декабря':
+            m = 12
```

Comment:
Is there another way to match date information? Or at least store mappings in the crawler configuration file than inside the code?